install/migrations/update_10.0.x_to_10.1.0/form.php

```diff
@@ -138,4 +137,33 @@
     $migration->changeField("glpi_forms_forms", "header", "header", "longtext");
     $migration->changeField("glpi_forms_sections", "description", "description", "longtext");
     $migration->changeField("glpi_forms_questions", "description", "description", "longtext");
+
+    // Deletion of subtype, use final type in `type` field instead
```

Comment:
```suggestion
    // Deletion of subtype, use final type in `type` field instead
    // TODO Delete this dev migration before releasing GLPI 10.1
```

Reply:
This code is already in the `TODO: Should be removed from the final release` scope a fews line above